pstatus-graphql-ktor/README.md

```diff
@@ -1,8 +1,34 @@
 # Overview
-This project is the processing status report sink.  It listens for messages on an Azure Service bus, validates the messages, and if validated persists them to CosmosDB.
+This project is the processing status service. The pstatus-graph-ktor service provides the following features:
+
+Ability to query for existing reports, uploads, upload status.
```

Comment:
need to mention about dead-letter reports as well

Reply:
need to mention about notifications as well

Reply:
Added descriptions for Dead letter Reports and Notifications.